kuma/settings/common.py

```diff
@@ -527,6 +530,7 @@ def enable_candidate_languages():
 
     # other
     'kuma.humans',
+    'redirect_urls',
```

Comment:
It is needed. This loads the machinery that then goes to find the `redirects.py` files in the other apps.